lib/email/EmailBase.js

```diff
@@ -83,6 +86,27 @@ class EmailBase {
     return emailSender;
   }
 
+  /**
+   * Returns the study request admin email address for this environment.  We
+   * redirect this to an internal MOVE list in non-production environments, to avoid
+   * spamming the Data Collection team with test requests!
+   */
+  static getRecipientStudyRequestAdmin() {
+    if (ENV === 'production') {
```

Comment:
this doesn't do what you think it should!  `ENV` here is the _application_ environment, not the _infrastructure_ environment - and the `production` _application_ environment refers to anything that's not local or CI / CD testing!

this should instead check `domain`.